client/package-lock.json

```diff
@@ -101,7 +101,7 @@
             }
         },
         "../iota-sdk/bindings/wasm": {
-            "name": "@iota/sdk-wasm-nova",
+            "name": "@iota/sdk-wasm",
```

Comment:
Is this correct? I believe it was changed automatically

Reply:
we shouldnt commit this, its the changes that `setup_nova.sh` do